io/seqio/fasta/fasta_example_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright ©2020 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package fasta_test
```

Comment:
There needs to be a space between the package decl and the copyright notice (not so much of an issue in this case, but in general, this would lead to the notice ending up in the package docs).